.travis.yml

```diff
@@ -97,16 +97,6 @@ jobs:
         # This could be removed once the ABI change warning does not show up by default
         BITCOIN_CONFIG="--enable-glibc-back-compat --enable-reduce-exports CXXFLAGS=-Wno-psabi"
 
-    - stage: test
-      name: 'Win32  [GOAL: deploy]  [no gui or functional tests]'
-      env: >-
-        HOST=i686-w64-mingw32
-        DPKG_ADD_ARCH="i386"
-        PACKAGES="python3 nsis g++-mingw-w64-i686 wine-binfmt wine32"
-        RUN_FUNCTIONAL_TESTS=false
-        GOAL="deploy"
-        BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests"
-
```

Comment:
Let's keep the Travis job...

Reply:
Why? I am not aware that this job ever failed and the win64 one didn't. Also, why would we want to waste resources on testing a target that we never ship?
I'd rather have freebsd tests than a win32 one.

Reply:
We ship source code.

Reply:
It is the responsibility of the user to run the tests. You could argue that many users don't run the tests on their target when they download the gitian binaries, but that doesn't apply here. If some users compile on their own, they need to run the tests themselves.

Reply:
Travis doesn't exist to run the tests. It exists to help us developers avoid doing things that will break the tests. (Running the tests is just how we accomplish that.)

Reply:
Either we support win32, test and ship it, or we don't. There is no in-between.

Reply:
I do not agree. We support plenty of things we do not recommend or ship binaries for.

Indeed, the best approach is to compile yourself, which itself falls outside the "ship" scope.

Reply:
FWIW I agree with @MarcoFalke here -- seems like not a great use of travis resources to explicitly test a platform that we're no longer interested in continuing to support (who will debug problems if they are found?).  On top of that, reducing the load on travis has benefits; I've noticed that the time I wait from updating a PR to having the travis jobs complete has ticked up recently.  So I'd rather we not spend those travis cycles on win32 unless it was an important platform to test on for some reason, and the decision to drop it strikes me as a statement that it's not.

Reply:
The decision not to provide binaries isn't necessarily a decision to drop support.

(That being said, *I* certainly am not interested in providing support, so if nobody else is either...)